embed.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ -z ${ISO_PATH+x} ]; then
+	echo "Please set ISO_PATH"
+	exit 1
+fi
+
+if [ -z ${IGNITION_PATH+x} ]; then
+	echo "Please set IGNITION_PATH"
+	exit 1
+fi
+
+if [ -z ${OUTPUT_PATH+x} ]; then
+	echo "Please set OUTPUT_PATH"
+	exit 1
+fi
+
+if [[ $(dirname $ISO_PATH) != $(dirname $IGNITION_PATH) ]]; then
+    echo $ISO_PATH and $IGNITION_PATH must be in the same directory
+    exit 1
+fi
+
+if [[ $(dirname $IGNITION_PATH) != $(dirname $OUTPUT_PATH) ]]; then
+    echo $IGNITION_PATH and $OUTPUT_PATH must be in the same directory
+    exit 1
+fi
```

Comment:
I think you can remove this and just add another volume mount to the `podman run` command, e.g.:
```
-v $(realpath $(dirname $ISO_PATH)):/data \
-v $(realpath $(dirname $IGNITION_PATH)):/ignition_data \
```
And update the `embed` param:
```
--ignition-file /ignition_data/$(basename $IGNITION_PATH)